knimin/lib/data_access.py

```diff
@@ -3312,6 +3365,118 @@ def get_sample_plate_list(self):
                 plates.append(row)
             return plates
 
+    def extract_sample_plates(self, sample_plate_ids, email, robot, kit_lot,
+                              tool, notes=None):
+        """Stores the extraction information for the given sample_plates
+
+        Parameters
+        ----------
+        sample_plate_ids : list of int
+            The sample plates being extracted
+        email : str
+            The email of the user preparing the extraction
+        robot : str
+            The name of the robot used for extraction
+        kit_lot : str
+            The kit lot used for extraction
+        tool : str
+            The name of the tool used for extraction
+        notes : str, optional
+            Notes added to the extracted plates
+        """
+        if not sample_plate_ids:
+            raise ValueError("Provide at least one sample plate to extract")
```

Comment:
This error is not mentioned in the docstring. Also, in what case a `sample_plate_ids` can be `[]`?